src/org/usfirst/frc/team1076/robot/subsystems/FrontBackMotors.java

```diff
@@ -27,15 +27,15 @@ public FrontBackMotors(SpeedController frontMotor, SpeedController backMotor) {
 	 * @param speed a number in the range -1 to 1 inclusive.
 	 */
 	public void setFrontSpeed(double speed) {
-		this.frontMotor.set(speed);
+		this.frontMotor.set(speed * 0.5);
```

Comment:
Yeah, I 100% agree with this, this was a hack while we were testing stuff before we left a bit late, so it's not the right solution.